pavelib/assets.py

```diff
@@ -240,17 +292,16 @@ def compile_sass(options):
         output_style = 'compressed'
 
     timing_info = []
-    system_sass_directories = applicable_sass_directories(systems)
-    all_sass_directories = applicable_sass_directories()
+    system_sass_directories = sass_source_directories(systems)
+    all_sass_directories = sass_lookup_directories()
     dry_run = tasks.environment.dry_run
-    for sass_dir in system_sass_directories:
+    for sass_dir, css_dir in system_sass_directories:
         start = datetime.now()
-        css_dir = sass_dir.parent / "css"
 
         if force:
             if dry_run:
                 tasks.environment.info("rm -rf {css_dir}/*.css".format(
-                    css_dir=css_dir,
+                    css_dir=css_dir or sass_dir.parent / "css",
                 ))
             else:
                 sh("rm -rf {css_dir}/*.css".format(css_dir=css_dir))
```

Comment:
Doesn't this need to handle ``css_dir`` being null, like the ``dry_run`` case does?